gitbutler-ui/src/lib/components/CommitCard.svelte

```diff
@@ -51,6 +51,15 @@
 		if (showFiles) loadFiles();
 	}
 
+	function onKeyup(e: KeyboardEvent) {
+		if (e.key === 'Enter' || e.key === ' ') {
+			showFiles = !showFiles;
+			if (showFiles) {
+				loadFiles();
+			}
+		}
+	}
+
```

Comment:
I was wondering if it would be neater if we renamed `onClick` to `toggleFiles` (or something to that effect) so we can then change `onKeyup` to

```
	function onKeyup(e: KeyboardEvent) {
		if (e.key === 'Enter' || e.key === ' ') {
			toggleFiles();
		}
	}
```

Reply:
Could we also use `==` instead of `===` to match the standard in the codebase?

Reply:
Good idea @Caleb-T-Owens thanks! And yeah, there should be `==` instead of `===`

Reply:
![telegram-cloud-photo-size-2-5463392310151730834-y](https://github.com/gitbutlerapp/gitbutler/assets/18498712/d2c5ad72-46b4-4252-8c07-09660b541318)